src/plugins/intel_gna/src/transformations/insert_identity_layer.cpp

```diff
@@ -231,3 +232,98 @@ bool IdentityCandidatesCleanup::run_on_model(const std::shared_ptr<ov::Model>& f
     }
     return false;
 }
+
+bool InsertIdentityForPrecAgnosticConcatInput::all_inputs_point_the_same_node(const shared_ptr<ov::Node>& node) {
```

Comment:
usually first work in function/method name is a verbe 
```suggestion
bool InsertIdentityForPrecAgnosticConcatInput::are_all_inputs_point_the_same_node(const shared_ptr<ov::Node>& node) {
```